### DISCUSSION TITLE
WARNING about DeepMD-kit： dp train

### DISCUSSION THREAD

**Discussion by walkjoker-c** [deepmodeling/deepmd-kit#992] [Q&A]
I configured a workstation with an RTX3090, installed centos Linux system and graphics card driver: NVIDIA-Linux-x86_64-470.63.01, I did not install any other software. I downloaded deepMD-kit and used the DP train command.

DPMD-kit version:                deepmd-kit-2.0.0.b4-cuda11.3_gpu-Linux-x86_64.sh

Program warning : 
WARNING:deepmd.train.run_options:Switch to serial execution due to lack of horovod module.

The training program is running normally. After Google, I learned that serial execution is inefficient. Do I get this program warning because I only have one GPU? What is the solution to switching to parallel execution mode

**Comment by njzjz** [comment]
You can see https://docs.deepmodeling.org/projects/deepmd/en/latest/train/parallel-training.html for details.
